### PR TITLE
Локализация docs/1.getting-started/4.assets.md (#28)

### PR DESCRIPTION
**Pull request by JonikUl**
### 🔗 Linked issue

Issue #28

### 📚 Description

Перевод [docs/1.getting-started/4.assets.md](https://nuxt-ru.vercel.app/docs/getting-started/assets)

### CODE REVIEW

**Review by Ibochkarev — APPROVED** [review]

File: docs/1.getting-started/4.assets.md [hunk 1]
```diff
@@ -1,59 +1,58 @@
 ---
-title: 'Assets'
-description: 'Nuxt offers two options for your assets.'
+title: 'Ассеты'
+description: 'Nuxt предлагает два варианта для ваших ассетов.'
 navigation.icon: i-ph-image-duotone
 ---
 
-Nuxt uses two directories to handle assets like stylesheets, fonts or images.
+Nuxt использует две директории для обработки таких ассетов, как таблицы стилей, шрифты и изображения.
```
Comment (Hekikai) [inline docs/1.getting-started/4.assets.md:7]:
```suggestion
Nuxt использует две директории для обработки таких ассетов как: таблицы стилей, шрифты и изображения.
```

File: docs/1.getting-started/4.assets.md [hunk 2]
```diff
@@ -1,59 +1,58 @@
...
+Вы можете получить файл в директории [`public/`](/docs/guide/directory-structure/public) из кода приложения или из браузера по корневому URL `/`.
 
 ### Пример
 
-For example, referencing an image file in the `public/img/` directory, available at the static URL `/img/nuxt.png`:
+Например, ссылка на файл изображения из директории `public/img/`, доступна по статическому URL `/img/nuxt.png`:
 
 ```vue [app.vue]
 <template>
   <img src="/img/nuxt.png" alt="Discover Nuxt 3" />
```
Comment (Hekikai) [inline docs/1.getting-started/4.assets.md:24]: Давай переведем альт

**Review by Hekikai — CHANGES_REQUESTED** [review]

File: docs/1.getting-started/4.assets.md (hunk 1 above)
Comment (JonikUl) [reply docs/1.getting-started/4.assets.md]: @Hekikai fixed

File: docs/1.getting-started/4.assets.md (hunk 2 above)
Comment (JonikUl) [reply docs/1.getting-started/4.assets.md]: @Hekikai fixed

**Review by Hekikai — APPROVED** [review]

**Pull request merged**